h5ad-to-arrow-manifest.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "pattern": "output/(.*)\.arrow",
```

Comment:
Backslash is a metacharacter in json, so I think you'll need to double it to get a single backslash to come through.